source/release-notes/4-2-0.rst

```diff
@@ -87,7 +95,7 @@ Dependency Updates
   - MariaDb4J to version ``3.1`` and MariaDB to ``11.3``
```

Comment:
Separate into two different lines (to highlight the MariaDB upgrade) and lead with MariaDB not MariaDB4j.